convert2rhel/actions/system_checks/eus.py

```diff
@@ -0,0 +1,53 @@
+# Copyright(C) 2016 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import datetime
+import logging
+
+from convert2rhel import actions, grub, systeminfo
+from convert2rhel.systeminfo import EUS_MINOR_VERSIONS, system_info
+from convert2rhel.toolopts import tool_opts
+
+
+logger = logging.getLogger(__name__)
+
+
+class EusSystemCheck(actions.Action):
+    id = "EUS"
+
+    def run(self):
+        """Inhibit the conversion when we are not able to handle UEFI."""
+        super(EusSystemCheck, self).run()
+
+        if "%s.%s" % (system_info.version.major, system_info.version.minor) == "8.8":
+            eus_release_date = EUS_MINOR_VERSIONS.get(
+                "%s.%s" % (system_info.version.major, system_info.version.minor), False
+            )
```

Comment:
```suggestion
        current_version = "%s.%s" % (system_info.version.major, system_info.version.minor) 
        if current_version == "8.8":
            eus_release_date = EUS_MINOR_VERSIONS.get(current_version, False)
```

Reply:
I'll change this, before I had current_version = "%s.%s" % (system_info.version.major, system_info.version.minor) in EUS_MINOR_VERSIONS so I would get a boolean return instead of the actual value